SQLScripts/20220722_AddClaimableAccountForUITests.sql

```diff
@@ -0,0 +1,19 @@
+﻿BEGIN TRY
+    BEGIN TRANSACTION
+        DECLARE @userID INT;
+
+        INSERT INTO Users (PrimaryEmail, PasswordHash, FirstName, LastName, JobGroupID, Active)

```

Comment:
This could result in different user ids in each person's db (eg. if they've incremented the sequence by adding their own user at some point before running this. It'd be nice to know the user id so that we can use it in tests.
The top ID in the Jenkins test db and in my db Users table is `281057`. Can we give this user the id `281058`? You just have to turn on IDENTITY_INSERT first and turn it off afterwards.

Reply:
I have set this to `281673`, as that is the value present in the new backup I uploaded to Sharepoint yesterday